lisa/tests/staging/sched_android.py

```diff
@@ -91,15 +82,11 @@ def __init__(self, res_dir, plat_info, test_bundles):
         self.test_bundles = test_bundles
 
     @classmethod
-    def from_target(cls, target:Target, res_dir:ArtifactPath=None,
+    def _from_target(cls, target:Target, res_dir:ArtifactPath=None,
             ftrace_coll:FtraceCollector=None) -> 'SchedTuneBase':
         """
         Creates a SchedTuneBase bundle from the target.
```

Comment:
We probably want to keep the warning from `from_target()`

Reply:
Actually the warning does not apply anymore, since the function actually exposed is  `RTATestBundle.from_target`. That said, the signature of the stub is actually wrong, since it copied it from `_from_target`, whereas in fact the actual method is `RTATestBundle.from_target`. I'll revisit that and push to the PR again once it's fixed.